README.md

```diff
@@ -11,6 +11,35 @@ It is backed by [hmpps-incident-reporting-api](https://github.com/ministryofjust
 
 The application needs a suite of services to work.
 
+### Using services in `dev` environment
+
+This is the easiest way to run and develop on your machine: by hooking into services that already exist
+in the `dev` environment.
+
+Run the application in development mode:
+
+* in separate shell sessions, for local development
+  * This will automatically restart it if server code or front-end assets are modified.
+
+```shell
+docker compose -f docker-compose.yml up
+npm run starvt:dev-local
+```
+
+* all in one, for preview:
+
+```shell
+docker compose -f docker-compose.yml --profile include-frontend up
+```
```

Comment:
there is already a "Using services in `dev` environment" section further down, but in fact i think you're trying to fill in the section "Using only local services" which is also further down but has a TODO placeholder.

also you mean `start` instead of `starvt`

Reply:
Yup corrected, now ready for review